### PR TITLE
style(ui): cleanup following page

### LINKED ISSUE #902: Add project display names to following tab
image

The Still Looking... section of a user's Following section should include the name of the project in addition to the URN. 

Figma [here](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=4147%3A7717)

@juliendonck - another one to pick up if you want.

### PR DESCRIPTION
**Pull request by juliendonck**
*before*
Screenshot 2020-09-15 at 15 55 08

*after*
Screenshot 2020-09-15 at 15 54 52

closes #902

### CODE REVIEW

File: ui/Screen/Profile/Tracking.svelte [hunk 1]
```diff
@@ -34,9 +34,27 @@
   ];
 
   const untracked = [
-    { urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k" },
-    { urn: "fjkldasjfkdlsajfio943we859043ikjioclesdjf" },
-    { urn: "fjkldasjfkdlsajfio39409340390we859043ikjioclesdjf" },
+    {
+      urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:38]: We should use the URN format that we settled on in: https://github.com/radicle-dev/radicle-upstream/issues/840.

```suggestion
      urn: "@hyndb5gs95gwtsf37tncz4ag3wqrg4ejw3qqga6x1srw9jp8jw59d6.git",
```

**Review by abbey-titcomb — DISMISSED** [review]

File: ui/Screen/Profile/Tracking.svelte [hunk 2]
```diff
@@ -34,9 +34,27 @@
...
+      metadata: {
+        name: "snickers",
+        description: "project description",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:41]: We won't have any other metadata available, aside from the name (which we only would get if it's included in the urn).

```suggestion
```

File: ui/Screen/Profile/Tracking.svelte [hunk 3]
```diff
@@ -34,9 +34,27 @@
...
+      },
+    },
+    {
+      urn: "fjkldasjfkdlsajfio943we859043ikjioclesdjf",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:45]:
```suggestion
      urn: "@hwd1yren6nte7ofh1sijz3tgc31cdmfb7zg7ya7gfgzwhhzgau8u13hkkjw.git",
```

File: ui/Screen/Profile/Tracking.svelte [hunk 4]
```diff
@@ -34,9 +34,27 @@
...
+      metadata: {
+        name: "marsbar",
+        description: "project description",
+      },
+    },
+    {
+      urn: "fjkldasjfkdlsajfio39409340390we859043ikjioclesdjf",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:52]:
```suggestion
      urn: "@qew1yren6nte7ofh1sijz3tgc31cdmfb7zg7ya7gfgzwhhzgau8u13hkfds.git",
```

File: ui/Screen/Profile/Tracking.svelte [hunk 5]
```diff
@@ -34,9 +34,27 @@
...
+      metadata: {
+        name: "nougati",
+        description: "project description",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:55]:
```suggestion
```

File: ui/Screen/Profile/Tracking.svelte [hunk 6]
```diff
@@ -34,9 +34,27 @@
...
+      metadata: {
+        name: "snickers",
+        description: "project description",
+      },
+    },
+    {
+      urn: "fjkldasjfkdlsajfio943we859043ikjioclesdjf",
+      metadata: {
+        name: "marsbar",
+        description: "project description",
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:48]:
```suggestion
```

File: ui/Screen/Profile/Tracking.svelte [hunk 7]
```diff
@@ -83,7 +106,10 @@
           <div
             class="undiscovered-project"
             out:fade|local={{ duration: FADE_DURATION }}>
-            <Urn urn={project.urn} showCopyOnlyOnHover />
+            <div>
+              <p class="project-name typo-text-bold">{project.metadata.name}</p>
+              <Urn urn={project.urn} showCopyOnlyOnHover />
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:111]: The string that should be copied to clipboard should be formatted like this:
`@hyndb5gs95gwtsf37tncz4ag3wqrg4ejw3qqga6x1srw9jp8jw59d6.git#rudolfs`.

https://github.com/radicle-dev/radicle-upstream/issues/840

Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte:108]: Decided to do this when we tackle #840

File: ui/Screen/Profile/Tracking.svelte (hunk 6 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

File: ui/Screen/Profile/Tracking.svelte (hunk 5 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

File: ui/Screen/Profile/Tracking.svelte (hunk 4 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

File: ui/Screen/Profile/Tracking.svelte (hunk 3 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

File: ui/Screen/Profile/Tracking.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

File: ui/Screen/Profile/Tracking.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/Screen/Profile/Tracking.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544

**Review by rudolfs — APPROVED** [review]

**Review by sarahscott — APPROVED** [review]

**Pull request merged**